src/C1T2X_OBU.py

```diff
@@ -115,7 +116,6 @@ def VANET_listening_thread():
 	global error
 
 	while not error and not vanet.error:
```

Comment:
We should probably use a mutex to guard access to `error` since both threads potentially modify it. Something like
```python
while not vanet.error:
    with mutex:
        if system_error:  # renamed from error to be more descriptive
            break

    # rest of the loop code
```